src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -32,10 +36,59 @@
     @MockBean
     private OcrDataValidator ocrDataValidator;
 
+    @MockBean
+    private AuthService authService;
+
+    @Test
+    void should_throw_unauthenticated_exception_when_auth_header_is_missing() throws Exception {
+        String requestBody = readResource("ocr-data/valid/valid-ocr-data.json");
+        given(authService.authenticate("")).willThrow(UnauthenticatedException.class);
+
+        mockMvc
+            .perform(
+                post("/validate-ocr-data")
+                    .header("ServiceAuthorization", "")
+                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                    .content(requestBody)
+            )
+            .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    void should_throw_unauthenticated_exception_when_auth_header_is_invalid() throws Exception {
+        String requestBody = readResource("ocr-data/valid/valid-ocr-data.json");
+        given(authService.authenticate("test-token")).willThrow(InvalidTokenException.class);
```

Comment:
same as above, tests in this class check that exceptions thrown by the injected service are translated to appropriate http status codes.

Reply:
I'll rename the tests

Reply:
renamed the tests